src/Entity/Client.php

```diff
@@ -95,12 +95,12 @@ public function setEmail(string $email): self
         return $this;
     }
 
-    public function getRole(): ?string
+    public function getRoles(): ?string
```

Comment:
Cet rôles est sensé retourner un tableau non?
Return [$this->rôle];

Reply:
Je pensais faire ça quand j'aurais traité l'authentification.
Done.